### PR TITLE
util orientation aliases

### PR DESCRIPTION
**Pull request by japalekhin**
added some alias functions for simplified calls when setting the game orientation

### CODE REVIEW

**Comment by erickzanardo** [comment]
LGTM, Just think we should add these methods to the docs

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 - Improve our mixin structure (breaking change)
 - Adds HasGameRef mixin
 - Fixes for ComposedComponent
+- Added no-parameter alias functions for setting the game's orientation.
+- Updated documentation on Util section.
```
Comment (erickzanardo) [inline CHANGELOG.md]: Not necessary to add entries for doc updates

File: doc/util.md [hunk 2]
```diff
@@ -12,14 +12,42 @@ It also differs from the default implementations provided (math.Point and ui.Off
...
+
+### `Flame.util.setOrientation()` and `Flame.util.setOrientations()`
+
+If a finer control on the allowed orientations is required (without having to deal with `SystemChrome` directly), `setOrientation` (accepts a single `DeviceOrientation` as a parameter) and `setOrientations` (accepts a `List<DeviceOrientation>` for possible orientations) can be used.
+
+### `Flame.util.initialDimensions()`
+
+Returns a `Future` with the dimension (`Size`) of the screen. This has to be done in a hacky way because of the reasons described in the code. If you are using `BaseGame`, you probably won't need to use these, as very `Component` will receive this information.
+
+**Note**: Call this function first thing in an `async`hronous `main` and `await` its value to avoid the "size bug" that affects certain devices when building for production.
```
Comment (erickzanardo) [inline doc/util.md]: Tried to look for the meaning of `hronous` and I could find, maybe this is a typo?

Comment (japalekhin) [reply doc/util.md]: it's "`async`hronous" :D

Comment (erickzanardo) [reply doc/util.md]: Lol, makes sense now, sorry for the confusion

File: CHANGELOG.md (hunk 1 above)
Comment (japalekhin) [reply CHANGELOG.md]: removed

**Pull request merged**